.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+	"cSpell.words": ["Disploy"]
+}
```

Comment:
Out of scope for this PR

Reply:
I mean while writing the guide, it was annoying that it kept coming up. So it’s kind of in scope.

Reply:
I think it's better to gitignore this entirely, for example what the [djs repo is doing](https://github.com/discordjs/discord.js/blob/main/.gitignore#L22)